app/components/windows/settings/OverlaySettings.tsx

```diff
@@ -99,6 +114,75 @@ export default class OverlaySettings extends Vue {
       });
   }
 
+  /**
+   * Convert a dual output scene collection to a vanilla scene collection
+   * @param assignToHorizontal Boolean for if the vertical sources should be assigned to the
+   * horizontal display or should be deleted
+   * @param exportOverlay Boolean for is the scene collection should be exported upon completion
+   */
+  async convertDualOutputCollection(
+    assignToHorizontal: boolean = false,
+    exportOverlay: boolean = false,
+  ) {
+    // confirm that the active scene collection is a dual output collection
+    if (
+      !this.sceneCollectionsService?.sceneNodeMaps ||
+      (this.sceneCollectionsService?.sceneNodeMaps &&
+        Object.values(this.sceneCollectionsService?.sceneNodeMaps).length === 0)
```

Comment:
Nitpick: would prefer checking `Object.keys` instead as runtime optimizations are most likely to be in place there.